core/platforms/packaging/bareos.spec

```diff
@@ -1065,16 +1065,9 @@ cmake  .. \
 # run unit tests
 pushd %{CMAKE_BUILDDIR}
 make clean
-REGRESS_DEBUG=1 ctest -j 10 -V -D Continuous || result=$?
-result=$?
-if [ $result -eq 1 ]; then
-  echo "ctest result $result is expected and OK"
-  exit 0
-else
-  echo "ctest result $result is not 1, ERROR"
-fi
-
 
+# run the tests and fail build if test fails
+REGRESS_DEBUG=1 ctest --label-exclude broken --no-compress-output -j 10 --repeat until-pass:2 -D Experimental || echo "ctest result:$?"
```

Comment:
* this doesn't actually fail, when ctest fails, or does it?
* can we use `--parallel` instead of `-j` and `--dashboard` instead of `-D`
* is "Experimental" the right Dashboard? It was "Continous" previously
```suggestion
# run the tests and fail build if test fails
REGRESS_DEBUG=1 ctest --label-exclude broken --no-compress-output --parallel 10 --repeat until-pass:2 --dashboard Continous
```

Reply:
It does not fail, that is correct. The Failing is implemented in the quality gate that analyses the output of ctest and then sets the build to failed. The ctest result unfortunately is not helpful to check if tests fail or were skipped.

Using **--parallel** and **--dashboard**  instead of the short switches make sense.

However, we might discuss about if we still need the cdash dashboard at all when we do the analysis inside of jenkins.